### DISCUSSION TITLE
Excluding templates from results?

### DISCUSSION THREAD

**Discussion by julie-za** [brianpetro/obsidian-smart-connections#293] [Q&A]
Hi, I heavily rely on the use of pre-defined templates. What I see is that the SmartConnections "result list" starts by showing all of these templates, then lists the notes that are based on the same template, it is only then that the list of the "atomic notes" starts (which is the most interesting part).

Am I not using it right - are there any settings I should try?

Also, if there is any list of tips and tricks (not too technical), I would appreciate a reference!

Thanks in advance for any help and advice!

**Comment by benvenker** [comment]
Have you tried using the folder_exclusions setting or the path_only setting? You can exclude entire folders of, say, template files, or all files with, e.g., "template" in the name.

**Comment by cjl12377** [comment]
hi @julie-za did you find out whether any solution worked for you? Having a similar problem as all my daily notes use the same template
